main/util/src/mill/util/Jvm.scala

```diff
@@ -454,6 +627,44 @@ object Jvm extends CoursierSupport {
     )(new Ctx.Home { override def home = os.home })
   }
 
+  def createClassLoader(
+      classPath: Iterable[os.Path],
+      sharedPrefixes: Iterable[String] = Seq(),
+      parent: java.lang.ClassLoader = null,
```

Comment:
Lets move `sharedPrefixes` next to `sharedLoader` so it's more obvious they're related